setup.py

```diff
@@ -31,12 +31,10 @@
         'Operating System :: POSIX :: Linux',
         'Topic :: Software Development :: Libraries :: Python Modules',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
Investigate comment on top of this file that says `# workaround for open() with encoding='' python2/3 compability`.

Reply:
Removed the `import os`. Looks like Py2 built-in `open` didn't support `encoding` parameter but now `io.open()` is just an alias for the built-in function.